source/NVDAObjects/UIA/wordDocument.py

```diff
@@ -89,6 +89,25 @@ def label(self):
 
 class WordDocumentTextInfo(UIATextInfo):
 
+	def _get_locationText(self):
+		rects=self._rangeObj.getBoundingRectangles()
+		left=rects[0]
+		top=rects[1]
+		# UIA has no good way yet to convert coordinates into user-configured distances such as inches or centimetres.
+		# Nor can it give us specific distances from the edge of a page.
+		# Therefore for now, get the screen coordinates, and if the word object model is available, use our legacy code to get the location text.
+		om=self.obj.WinwordWindowObject
+		if not om:
+			raise NotImplementedError

```

Comment:
Woops, missed this one. This is actually not caught in script_navigatorObject_currentDimensions. You do want to return super here instead.